security/firely-auth/firely-auth-settings.rst

```diff
@@ -17,6 +17,8 @@ Just like Firely Server itself, Firely Auth features a hierarchy of settings fil
 
 Unsure how to name your variables? This works the same as with :ref:`configure_envvar`.
 
+Providing a folder for your settings files also works the same as with :ref:`configure_settings_path`, but then using the environment variable ``FIRELY_AUTH_PATH_TO_SETTINGS``.
+
```

Comment:
I would really specify here that this is handy for deployments in e.g. Azure or AWS that would not work otherwise. The comment comes a bit out of the blue now. It would also be good to have a link from the docker deployment part of firely-auth-deploy.rst to this page,  with something like:
"If you want to let docker pick up your *.instance.json files, this can be done by using an environment variable<pointing to this page>"
Maybe even add a bit of code here for the docker deployment picking up the environment variable, such as:

docker run --rm -p 5100:5100 -p 5101:5101 --name firely.auth --user=firely:firely -v /${PWD}/firely-auth-license.json:/app/firely-auth-license.json  -v /${PWD}/config:/app/settings -e FIRELY_AUTH_PATH_TO_SETTINGS=/app/settings firely.azurecr.io/firely/auth:latest